js/components/interface/graph-visualization/forces/collide.js

```diff
@@ -0,0 +1,156 @@
+import { binarytree } from "d3-binarytree";
```

Comment:
What about a description of what's inside here? Looks like it's complex stuff!

Reply:
I'm starting to understand these are all forces we are applying, seems really important anyway to understand what they're there for